### DISCUSSION TITLE
Colab file

### DISCUSSION THREAD

**Discussion by fciaf420** [kyleskom/NBA-Machine-Learning-Sports-Betting#216] [Q&A]
If i run the colab file and get a successful bootstrap message, does that encompass pulling the most up todate data ?

**Comment by chriseling** [comment]
It loads the code to run the trained model.  The model was trained with
data up through part of this season.  Then when you run the command, it
will pull the latest odds and run the model for you.
Best,
Chris

**Comment by fciaf420** [comment]
SO the data is not up to date then? Sent from my iPhoneOn Apr 24, 2023, at 3:01 PM, Chris Ling ***@***.***> wrote:﻿
It loads the code to run the trained model.  The model was trained with
data up through part of this season.  Then when you run the command, it
will pull the latest odds and run the model for you.
Best,
Chris

**Comment by kyleskom** [comment]
The data the model is trained on does not need to be updated every day as thats now how models work. We do grab updated data for the teams every say that is used to make predictions with the model.
